website/templates/profile/notifications.mako

```diff
@@ -44,13 +45,23 @@
                     <!-- Flashed Messages -->
                     <div data-bind="html: message, attr: {class: messageClass}"></div>
             </div><!--view model scope ends -->
-            </div>
+        </div>
+        <div class="panel panel-default">
+            <div class="panel-heading"><h3 class="panel-title">Configure Notification Preferences</h3></div>
+                <form id="selectNotifications" class="osf-treebeard-minimal">
+                    <div id="grid"></div>
+                    <div class="help-block" style="padding-left: 15px">
+                            <p id="configureNotificationsMessage"></p>
+                    </div>
+                </form>
```

Comment:
While it's not consistent with other forms on the page, the advantage of having the settings update automatically is that the user won't loose these changes if they happen to navigate away.